gitlab-examples/gitlab-pipeline/README.md

```diff
@@ -7,11 +7,12 @@ It than uses JFrog Xray to scan the build for OSS vulnerabilities.
 1. Create a GitLab project
 2. Import a Gradle repository into the project (I used https://github.com/spring-projects/spring-petclinic)
 3. Configure the following variables in GitLab (Settings->CI/CD->Variables):
-   1. ARTIFACTORY_SERVER_ID - a unique server ID used to reference the server configuration from within the pipeline code
-   2. ARTIFACTORY_URL - the Artifactory server used
-   3. ARTIFACTORY_USER - the user name used to access Artifactory
-   4. ARTIFACTORY_PASS - a password used to validate the user
-   5. ARTIFCATORY_REPO - a virtual repository used to upload and download artifacts from Artifactory
+   1. JFROG_PLATFORM_ID - a unique server ID used to reference the server configuration from within the pipeline code
```

Comment:
Are you sure that the server ID is really needed here? It is an optional parameter to `jf config add`.

Reply:
I think is is a good practice to have server ID, this is done in most of our examples.